src/main/java/org/justjava/gymcore/config/OpenApiConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.justjava.gymcore.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+public class OpenApiConfig {
+
+    @Bean
+    public OpenAPI customOpenAPI() {
+        return new OpenAPI()
+                // 1) Set up your Info block
+                .info(new Info()
+                        .title("Gym Core REST API")
+                        .version("v1.0")
+                        .description("Manage gym memberships, classes, bookings, and users."))
+                // 2) Override the default server URL
+                .servers(List.of(
+                        new Server()
+                                .url("http://localhost:8080")
```

Comment:
this url may generate issue if we put this behind a reverse proxy or AWS, better to not to give static url here.